plugin.jest.js

```diff
@@ -932,6 +933,7 @@ describe('CspHtmlWebpackPlugin', () => {
         expect(processFn).toHaveBeenCalledWith(
           index1BuiltPolicy,
           expect.anything(),
+          expect.anything(),
```

Comment:
Could we also add a test checking that we can add the policy to a conf file, as recommended in the readme? If should go in the `Custom process function` section

We should be able to write to a file, and the read the file generated and expect it to match the policy as written - the callback in `webpackCompile` has access to the `fileSystem` to allow us to do this

Reply:
Very good idea, I've just added this test.